src/main/kotlin/uk/gov/dwp/dataworks/aws/AwsCommunicator.kt

```diff
@@ -540,4 +549,23 @@ class AwsCommunicator {
             return false
         }
     }
+
+
+    fun rdsExecuteStatement(sql: String, parameters: Collection<SqlParameter>? = null): ExecuteStatementResponse {
+        try{
+            val executeStatementRequest = ExecuteStatementRequest
+                .builder()
+                .secretArn(configurationResolver.getStringConfig(ConfigKey.RDS_CREDENTIALS_SECRET_ARN))
+                .resourceArn(configurationResolver.getStringConfig(ConfigKey.RDS_CLUSTER_ARN))
+                .database(configurationResolver.getStringConfig(ConfigKey.RDS_DATABASE_NAME))
+                .parameters(parameters)
+                .sql(sql)
+
+            return awsClients.rdsDataClient.executeStatement(executeStatementRequest.build())
+        } catch (e: SystemArgumentException){
```

Comment:
[The Docs](https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/services/rdsdata/RdsDataClient.html#executeStatement--) don't show this as an Exception thrown by this call - is this what has been thrown in the tests?

Reply:
SystemArgumentException is thrown by the configuration resolver when a value is missing